src/app/txid/[txid]/TxDetails/BlockHeight.tsx

```diff
@@ -11,6 +11,8 @@ import { KeyValueHorizontal } from '../../../common/components/KeyValueHorizonta
 import { Value } from '../../../common/components/Value';
 import { isInMempool, isInMicroblock } from '../utils';
 
+import { useMediaQuery } from '@chakra-ui/react'
```

Comment:
Could you please wrap `useMediaQuery` under /ui directory similar to the rest of ChakraUI components instead of using it directly?